code/game/objects/structures/crates_lockers/closets/statue.dm

```diff
@@ -17,7 +17,7 @@
 	if(ishuman(L) || iscorgi(L))
 		if(L.buckled)
 			L.buckled = 0
-			L.anchored = 0
+			L.anchored = FALSE
 		L.forceMove(src)
 		ADD_TRAIT(L, TRAIT_MUTE, STATUE_MUTE)
 		max_integrity = L.health + 100 //stoning damaged mobs will result in easier to shatter statues
```

Comment:
this is abhorrent.

Reply:
it won't let me comment on it thought but would you change the `timer` variable to use the `MINUTES` define.

Reply:
I assume SECONDS is more appropriate here...